.circleci/config.yml

```diff
@@ -358,14 +360,10 @@ workflows:
 
   rerun-from-failed:
     when:
-      condition:
-        equal: ["<< pipeline.schedule.name >>", "rerun-from-failed"]
+      equal: [rerun-from-failed, << pipeline.schedule.name >>]
```

Comment:
based on https://circleci.com/docs/schedule-pipelines-with-multiple-workflows/#schedule-using-built-in-pipeline-values